### PR TITLE
add learning objectives in readme

### PR DESCRIPTION
**Pull request by tomvannuenen**
This PR adds a section on learning objectives in the README.md. It also includes a sentence on aspects that are not covered in the workshop, so participants know better what to expect.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,14 +5,19 @@
 
 This repository contains the materials for D-Lab [WorkshopName] workshop. Prior experience with [PriorWorkshopWithLink] is assumed.
 
-## Workshop Goals
+## Learning Objectives
 
 In this workshop, we provide [WorkshopGoal]. First, we’ll cover [Aim1]. Then, we’ll explore [Aim2]. 
 
+After this workshop, you will be able to [LearningObjective1]. You will [LearningObjective2] and [LearningObjective3].
```
Comment (pssachdeva) [inline README.md]: Can you convert the learning objectives and not-learning objectives both to lists?

File: README.md [hunk 2]
```diff
@@ -5,14 +5,19 @@
...
+
+This workshop does not cover [NotCovered1] and [NotCovered2].
+
 Provide additional detail as required. This section should not be more than 2 paragraphs.
```
Comment (pssachdeva) [inline README.md]: Do we still need this if the learning objectives are explicit?

File: README.md [hunk 3]
```diff
@@ -5,14 +5,19 @@
 
 This repository contains the materials for D-Lab [WorkshopName] workshop. Prior experience with [PriorWorkshopWithLink] is assumed.
 
-## Workshop Goals
+## Learning Objectives
 
 In this workshop, we provide [WorkshopGoal]. First, we’ll cover [Aim1]. Then, we’ll explore [Aim2].
```
Comment (pssachdeva) [inline README.md]: What is the difference between Aims and Learning Objectives?

**Review by pssachdeva — CHANGES_REQUESTED** [review]
Left some comments.

File: README.md (hunk 3 above)
Comment (tomvannuenen) [reply README.md]: I see Workshop Goals as a quick summary paragraph with goals and what you need. The Learning Objectives give more in-depth info on what to expect and not expect.

**Review by pssachdeva — APPROVED** [review]

**Comment by pssachdeva** [comment]
LGTM, could you squash and merge when ready?

**Pull request merged**